app/models/agents/google_calendar_publish_agent.rb

```diff
@@ -1,4 +1,5 @@
 require 'json'
+require 'google_calendar'
```

Comment:
Can we require this in the `receive` method so it's not needed when `gem_dependency_check` is false?